pages/index.tsx

```diff
@@ -1,43 +1,395 @@
+							'dolor sit amet, consectetur adipiscing elit. Integer gravida orci lacus, vel convallis enim posuere ac.'
+						}
+						link={<Link href={'#'}>See more</Link>}
+						small
+						withBackground
+					/>
+					<InlineMessage.Warning
+						title={'Lorem ipsum'}
+						description={
+							'dolor sit amet, consectetur adipiscing elit. Integer gravida orci lacus, vel convallis enim posuere ac.'
+						}
+						link={<Link href={'#'}>See more</Link>}
+						small
+						withBackground
+					/>
+					<InlineMessage.Destructive
+						title={'Lorem ipsum'}
+						description={
+							'dolor sit amet, consectetur adipiscing elit. Integer gravida orci lacus, vel convallis enim posuere ac.'
+						}
+						link={<Link href={'#'}>See more</Link>}
+						small
+						withBackground
+					/>
+				</section>
+				<section>
+					<h2>Links</h2>
+					<Link href="#">Link</Link>
+				</section>
+				<section>
+					<h2>Layout</h2>
+					<h3>Stack</h3>
+					<Layout header={'header'} footer={'footer'}>
+						main
+					</Layout>
+				</section>
+				<section>
+					<h2>Loading</h2>
+					<Loading />
+				</section>
+				<section>
+					<h2>Menu</h2>
+					<Menu>
+						<Menu.Item iconBefore="talend-stop" href="#" active>
+							Item A
+						</Menu.Item>
+						<Menu.Item iconBefore="talend-stop" href="#">
+							Item B
+						</Menu.Item>
+						<Menu.Item iconBefore="talend-stop" href="#">
+							Item C
+						</Menu.Item>
+					</Menu>
+				</section>
+				<section>
+					<h2>Modal</h2>
+					<Modal
+						disclosure={<Button.Primary>Send invitation email</Button.Primary>}
+						title="Send invitation email"
+					>
+						Invitation email will be send to this address: <strong>test@email.com</strong>
+						<br />
+						Do you want to send this email now?
+					</Modal>
+				</section>
+				<section>
+					<h2>Popover</h2>
+					<Popover
+						aria-label="Custom popover"
+						disclosure={<Button.Primary>Open popover</Button.Primary>}
+					>
+						Custom popover
+					</Popover>
+				</section>
+				<section>
+					<h2>Skeleton</h2>
+					<Skeleton />
+				</section>
+				<section>
+					<h2>Switch</h2>
+					<Switch value="a" values={['a', 'b', 'c']} />
+				</section>
+				<section>
+					<h2>Tabs</h2>
+					<Tabs.TabList>
+						<Tabs.Tab>Tab 1</Tabs.Tab>
+						<Tabs.Tab>Tab 2</Tabs.Tab>
+						<Tabs.Tab>Tab 3</Tabs.Tab>
+					</Tabs.TabList>
+				</section>
+				<section>
+					<h2>Tags</h2>
+					<Tag.Success>Tag</Tag.Success>
+					<Tag.Destructive>Tag</Tag.Destructive>
+					<Tag.Information>Tag</Tag.Information>
+					<Tag.Warning>Tag</Tag.Warning>
+				</section>
+				<section>
+					<h2>Toggles</h2>
+					<Toggle icon="talend-datagrid">Toggle</Toggle>
+				</section>
+				<section>
+					<h2>Tooltips</h2>
+					<Tooltip title="tooltip title">
+						<Button>Button</Button>
+					</Tooltip>
+				</section>
+				<section>
+					<h2>Visually Hidden</h2>
+					<VisuallyHidden>this content is visually hidden</VisuallyHidden>
```

Comment:
This doesn't show anything in the showcase :D

Reply:
WDYT?
```suggestion
					This content is visible<VisuallyHidden> and, this, is visually hidden</VisuallyHidden>!
```